app/assets/stylesheets/layout.scss

```diff
@@ -302,6 +302,7 @@ td, .table-list li { font-size: 14px; }
 .user-right-content-box { width: calc(100% - 200px); }
 .icon-right-content-box { width: calc(100% - 150px); }
 .character-right-content-box { width: calc(100% - 175px); }
+#user-avatar { text-align: center !important; background-color: #B0BBA9; }
```

Comment:
Is this the right background color in the other layouts?

Reply:
I think I have `#user-avatar` on the 'things replacing green' list in them. May put that into icon-boxes on it anyway though, for ease of use.